### ISSUE TITLE
Hide web elements

### ISSUE THREAD

**Issue by javidominguez** [accessolutions/WebAccessForNVDA#27]
Sometimes on a web page there are elements that are repetitive and superfluous, that do not provide information and make reading difficult. In the same way that the role of an element can be transformed, it would be interesting to be able to hide the elements we want skip.

**Comment by frederic-brugnot** [comment]
Yes, that's a good idea. This feature would be very useful for making many web pages easier to read by removing distracting elements such as some advertisements...

For technical reasons, WebAccess transformations are not the right method to achieve this because they currently only allow modifications of certain roles but not the actual content of the page in navigation mode.

Instead, I suggest adding a "hidden" property to a zone-type rule so that this zone is excluded from navigation with NVDA.
This would have a bit of the opposite effect of the functionality of a zone that limits navigation to within it. In the case of a hidden zone, therefore, it would be the opposite operation.

However, before this feature can be developed, we will first need to develop another feature that we will implement, non-sequential zones, which will be specified in another ticket.
